ComputePipeline/src/ComputePipeline.jl

```diff
@@ -6,6 +6,18 @@ using Base: RefValue
 
 abstract type AbstractEdge end
 
+"""
+    struct Computed ... end
```

Comment:
```suggestion
    Computed
```

or

```suggestion
    struct Computed
```

are the "standard" ways of doing this IIRC

Reply:
Fixed, thanks